implementation/src/main/java/io/smallrye/mutiny/operators/AbstractUni.java

```diff
@@ -106,9 +106,14 @@
                 new UniRunSubscribeOn<>(this, executor));
     }
 
+    @Override
+    public io.smallrye.mutiny.groups.UniMemoize memoize() {
```

Comment:
just for my understanding: why the FQ name instead of an import statement?

Reply:
It's because we had 2 `UniMemoize` classes (operator and group) in a previous commit, I'll fix that